rlp/src/main/java/net/consensys/cava/rlp/RLPException.java

```diff
@@ -19,4 +19,12 @@
   public RLPException(String message) {
     super(message);
   }
+
+  public RLPException(Throwable cause) {
+    super(cause);
+  }
+
+  public RLPException(String message, Throwable cause) {
+    super(message, cause);
```

Comment:
Not sure this change relates to readByteArray.

Reply:
It doesn't, but is mentioned in the PR and commit.